LottieSample/src/main/kotlin/com/airbnb/lottie/samples/PlayerFragment.kt

```diff
@@ -522,4 +525,22 @@ class PlayerFragment : Fragment() {
             }
         }
     }
+
+    /* Calculate and returns current and total frames along wih current and total seconds */
+    private fun updateFramesAndDurationLabel(animation: LottieAnimationView): String {
+
+        var label = ""
+
+        // current frame out of total frames
```

Comment:
@markstonehouse 
I think this method is confusing to read as is. I would suggest:
1) Most of the comments are pretty redundant. This code should be simple enough 
that it is self-documenting.
2) I would suggest something like this:
```
val currentFrame = ...
val totalFrames = ...
val progress = ...
return "$currentFrame/$totalFrames\n$progress%"
```

I think it would read more clearly.